microsoft/testsuites/dpdk/dpdksuite.py

```diff
@@ -42,11 +54,11 @@ def check_dpdk_build(self, node: Node, log: Logger) -> None:
 
         # grab a nic and run testpmd
         test_nic = node_nic_info.get_nic(node_nic_info.get_upper_nics()[-1])
-        vdev_type = "net_vdev_netvsc0"
-        testpmd_include_str = test_nic.testpmd_include(vdev_type)
+
+        testpmd_include_str = test_nic.testpmd_include(VDEV_TYPE)
         testpmd_cmd = testpmd._generate_testpmd_command(testpmd_include_str, "txonly")
```

Comment:
the `_generate_testpmd_command` should be a public method, because it's accessed out of the class.